doc/can-observable-array.md

```diff
@@ -104,7 +104,7 @@ with `new` to create observable lists.  For example:
 
 ```js
 import { ObservableArray } from "can/everything";
-const list = new ObservableArray( [ "a", "b", "c" ] );
+const list = new ObservableArray(...[ "a", "b", "c" ]);
```

Comment:
can we just show `new ObservableArray( "a", "b", "c" )`?

Reply:
@phillipskevin Do you want this for all examples including:
```js
const strings = new StringList(...[ 1, new Date( 1475370478173 ), false ]);
```

Reply:
Yeah, I think it's best to do it this way for all the examples.